### PR TITLE
Fix: Activity feed layout

### PR DESCRIPTION
**Pull request by priyajeet**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @priyajeet has signed the CLA. Thanks for the pull request!

**Comment by boxcla** [comment]
# Webpack Bundle Size Github Bot

| Bundle        | Diff | Percentage Diff | Before | After |
| ------------- |:----:|:---------------:|:------:|:-----:|
| Total | 0 B | 0.00% | 0 B | 0 B |

Compared against previously merged PR #389

**Powered by [webpack-bundle-size-github-bot](https://github.com/wenboyu2/webpack-bundle-size-github-bot/)**

File: src/components/ContentSidebar/ActivitySidebar.js [hunk 1]
```diff
@@ -56,7 +56,7 @@ const ActivitySidebar = ({
     getAvatarUrl,
     getUserProfileUrl
 }: Props) => (
-    <SidebarContent title={<FormattedMessage {...messages.sidebarActivityTitle} />}>
+    <SidebarContent hasScrollbar={false} title={<FormattedMessage {...messages.sidebarActivityTitle} />}>
```
Comment (jeremypress) [inline src/components/ContentSidebar/ActivitySidebar.js]: Why are we setting this to false?

Comment (priyajeet) [reply src/components/ContentSidebar/ActivitySidebar.js]: Since activity feed has its own internal scrollbar, not at the sideabar content level

**Pull request merged**